ui/components/ui/tabs/tabs.types.ts

```diff
@@ -2,21 +2,15 @@ import React from 'react';
 import { type BoxProps, type TextProps } from '@metamask/design-system-react';
 
 export type TabProps<TKey extends string = string> = Omit<
-  BoxProps,
+  TextProps,
   'children' | 'onClick' | 'ref'
 > & {
-  /** CSS class name for the active tab state */
-  activeClassName?: string;
-  /** CSS class name for the tab button */
-  buttonClassName?: string;
```

Comment:
These props weren't really being used effectively and aren't necessary so have removed all instances to make the component simpler 

<img width="309" height="254" alt="Screenshot 2025-09-18 at 2 16 53 PM" src="https://github.com/user-attachments/assets/bacccee7-60f7-4151-ac3d-8bef45b3fdd2" /><img width="308" height="253" alt="Screenshot 2025-09-18 at 2 17 03 PM" src="https://github.com/user-attachments/assets/64bf33f1-1923-49a8-8d75-33abbc717661" />